src/containers/EntryPage.js

```diff
@@ -110,7 +110,7 @@ class EntryPage extends React.Component {
     const { deleteEntry, entry, collection } = this.props;
     const slug = entry.get('slug');
     setTimeout(() => {
-      deleteEntry(collection, slug).then(() => this.handleCloseEntry());
+      deleteEntry(collection, slug).then(() => this.handleCloseEntry()).catch(() => {/* Don't do anything, the error will be shown by the action. */});
```

Comment:
I think the error here should continue to propagate, rather than be caught by the empty `catch` block, unless there's a particular reason you caught this here.